src/components/simulation/SimulationHistoryDataset.svelte

```diff
@@ -296,13 +298,17 @@
   }
 
   .simulation-dataset-status-cancel {
-    background: var(--st-gray-15);
     border-radius: 16px;
-    color: var(--st-gray-70);
+    color: var(--st-gray-60);
+  }
+
+  .simulation-dataset-status-cancel:hover:disabled {
+    cursor: not-allowed;
   }
 
-  .simulation-dataset-status-cancel:hover {
+  .simulation-dataset-status-cancel:hover:enabled {
     background: var(--st-gray-30);
+    color: var(--st-gray-80);
```

Comment:
Hmm, does that mean I should add `background` and `color` to `.simulation-dataset-status-cancel:hover:disabled` instead of just `cursor: not-allowed;`?

Reply:
Simply swapping the order and dropping the `:enabled` gets me a combination of the two, rather than just the cursor